package.json

```diff
@@ -83,11 +83,11 @@
     "style-loader": "^0.20.2",
     "stylelint": "^9.1.1",
     "stylelint-scss": "^2.4.0",
-    "stylelint-webpack-plugin": "^0.10.3",
+    "stylelint-webpack-plugin": "git://github.com/waterfoul/stylelint-webpack-plugin.git#0f46e8f",
```

Comment:
using this patch until the stylelint-webpack-plugin merges [this PR](https://github.com/JaKXz/stylelint-webpack-plugin/pull/139)